examples/NatSeq/NatSeqExample.dfy

```diff
@@ -0,0 +1,43 @@
+// RUN: %dafny /compile:0 /noNLarith "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT 
+*******************************************************************************/
+
+include "../../src/Collections/Sequences/NatSeqConversions.dfy"
+
+module NatSeqExample {
+
+  import opened Uint8_32
```

Comment:
I think this is a really cool and clear use case for module refinement in Dafny. Could you also add an example here of refining the general `NatSeqConversions` module with some less common small and large widths, just to highlight that users can do that themselves if they need to?